commands/auth.go

```diff
@@ -234,7 +234,7 @@ func currentCmdF(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	fmt.Printf("\n  found credentials for %v: %v @ %v\n\n", credentials.Name, credentials.Username, credentials.InstanceUrl)
+	fmt.Printf("\n  found credentials for %s: %s @ %s\n\n", credentials.Name, credentials.Username, credentials.InstanceUrl)
```

Comment:
Nit: Maybe also use `%q` here?

Reply:
Sure, I was going to ask it :)

Reply:
Well, you are touching the code here, so I would opt for also fixing this issue here. Other issues can be fixed in a separate PR.